src/libs/validationHandler.ts

```diff
@@ -21,15 +21,15 @@ export default ( config ) => ( req: Request, res: Response, next: NextFunction
       // Checking for required
       console.log('values is' , values);
       if (obj.required) {
-          if (isNull(paramvalue)) {
+          if (!isNull(paramvalue)) {
```

Comment:
Remove ! from here.
You have an issue with your find function above you need to add "!" there to return the first non-empty value.

`const paramvalue = values.find((val) => {  return !isNull(val); });	      });`